docs/source/conf.py

```diff
@@ -48,7 +48,8 @@
 # a list of builtin themes.
 #
 html_theme = 'sphinxdoc'
-html_logo = 'pytentiostat_logo.png'
+html_logo = 'pytentiostat_logo_10212019.png'
+html_favicon = 'pytentiostat_favicon_10212019.png'
```

Comment:
Maybe should rename more simple and direct.

Reply:
In my mind these were pretty direct. I guess we can get rid of pytentiostat, although I would like to keep the date so that there is some way to track the version. Open to other suggestions, although there should be a uniform naming system for these figures so that they are easier to track.